lib/CL/devices/printf_base.c

```diff
@@ -428,12 +427,12 @@ unsigned
 __pocl_printf_puts_ljust (param_t *p,
                           const char *string,
                           unsigned width,
-                          int max_width)
+                          int inp_max_width)
```

Comment:
Not sure what "inp" means. Does it mean "input"? Should it be named as "input_max_width"?

Reply:
I've changed the internal unsigned `max_width` to `max_width_u`. This should clear up the confusion.